lib/index.js

```diff
@@ -1672,7 +1672,9 @@ Stream.prototype.write = function (x) {
 /**
  * Forks a stream, allowing you to add additional consumers with shared
  * back-pressure. A stream forked to multiple consumers will only pull values
```

Comment:
what about: 
` A stream forked to multiple consumers will only pull values, ONE AT A TIME, from its source as fast as the slowest consumer can handle them. `

Reply:
Added.